src/lib/browse/MrnLayer.svelte

```diff
@@ -28,17 +30,13 @@
     sourceLayer: name,
     color,
     width: 7,
+    opacity: hoveredToggle(0.5, 1.0),
```

Comment:
For all these contextual layers, change the opacity when hovering on something to emphasize what it is